mods/cnc/rules/aircraft.yaml

```diff
@@ -68,6 +68,7 @@ HELI:
 	Aircraft:
 		TurnSpeed: 7
 		Speed: 180
+		LandableTerrainTypes: Clear,Rough,Road,Ore,Beach,Tiberium,BlueTiberium
```

Comment:
All aircraft in TD except transport helis have `LandWhenIdle: false` and can only land on helipads, and the terrain type shouldn't matter there, so these yaml changes should be unnecessary.

Reply:
Unfortunately, that is not true. In order to fix #2018 aircraft will have to explicitly check whether they can land on a given cell even when called from ReturnToBase. when calling CanLand they will then also check the underlying terrain. I admit this is a bit hacky. There is probably a better solution to this.

Even so, I think helicopters should at least in theory be able to land outside of the helipad even if currently there is no way to order them to do so. They could in the original and conceptually it makes sense.

Reply:
Ok. I've added an extra check in `CanLand` so this hack is not necessary anymore. Removed `LandableTerrainTypes` from non-vtol craft. I've left it in for helicopters. I guess that's just a matter of taste. I will remove it if you find it really objectionable.

Reply:
Well, 'objectionable' sounds a bit too harsh, we just try to avoid keeping lines around that won't have any effect on that actor.
That being said, as a compromise you could move this to the `^Helicopter` default instead.